pkg/client/cache/listwatch.go

```diff
@@ -23,33 +23,37 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/watch"
 )
 
+// ListFunc knows how to list resources
+type ListFunc func() (runtime.Object, error)
+
+// WatchFunc knows how to watch resources
+type WatchFunc func(resourceVersion string) (watch.Interface, error)
+
 // ListWatch knows how to list and watch a set of apiserver resources.  It satisfies the ListerWatcher interface.
-// It is a convenience function for users of NewReflector, etc.  Client must not be nil.
+// It is a convenience function for users of NewReflector, etc.
+// ListFunc and WatchFunc must not be nil
 type ListWatch struct {
-	Client        *client.Client
-	FieldSelector labels.Selector
-	Resource      string
-	Namespace     string
+	ListFunc  ListFunc
+	WatchFunc WatchFunc
```

Comment:
Format looks good to me. It just has the two member fields and they are aligned.